packages/core/helper-plugin/lib/src/components/ContentBox/index.js

```diff
@@ -15,25 +16,43 @@ const TypographyWordBreak = styled(Typography)`
   word-break: break-all;
 `;
 
-const ContentBox = ({ title, subtitle, icon, iconBackground, endAction, titleEllipsis }) => {
+const ContentBox = ({
+  title,
+  subtitle,
+  icon,
+  iconBackground,
+  endAction,
+  titleEllipsis,
+  customIcon,
+  decorator,
+}) => {
   if (title.length > 70 && titleEllipsis) {
     title = `${title.substring(0, 70)}...`;
   }
 
   return (
-    <Flex shadow="tableShadow" hasRadius padding={6} background="neutral0">
-      <IconWrapper background={iconBackground} hasRadius padding={3}>
-        {icon}
-      </IconWrapper>
+    <Flex shadow="tableShadow" hasRadius padding={6} background="neutral0" position="relative">
+      {customIcon || (
+        <IconWrapper background={iconBackground} hasRadius padding={3}>
+          {icon}
+        </IconWrapper>
+      )}
       <Stack spacing={endAction ? 0 : 1}>
         <Flex>
           <TypographyWordBreak fontWeight="semiBold" variant="pi">
             {title}
           </TypographyWordBreak>
           {endAction}
         </Flex>
-        <Typography textColor="neutral600">{subtitle}</Typography>
+        {decorator ? (
+          <Box width="80%">
+            <Typography textColor="neutral600">{subtitle}</Typography>
+          </Box>
+        ) : (
+          <Typography textColor="neutral600">{subtitle}</Typography>
+        )}
       </Stack>
```

Comment:
I agree with Josh, I think using `children` here would be a good way to make this more flexible and avoid needing 1 more custom prop